features/steps.js

```diff
@@ -1,18 +1,41 @@
 const { Given, When, Then } = require('cucumber')
-const Sandbox = require('./sandbox')
+const ClientSandbox = require('./client-sandbox')
+const ContributorSandbox = require('./contributor-sandbox')
+const assert = require('assert').strict
 
-Given('Compensated is configured with a clean {paymentGateway} account', function (paymentGateway) {
-  this.sandbox = new Sandbox(paymentGateway)
+Given(
+  'Compensated is configured with a clean {paymentGateway} account',
+  function (paymentGateway) {
+    return (this.clientSandbox = new ClientSandbox(paymentGateway))
+  }
+)
+
+Given('the following language test matrix:', function (languageMatrix) {
+  this.languageMatrix = languageMatrix.hashes()
 })
 
 Given('there is a compensated.json with the following data:', function (json) {
-  return this.sandbox.createFileSync('compensated.json', json)
+  return this.clientSandbox.createFileSync('compensated.json', json)
 })
 
 When('I run `compensated apply`', function () {
-  return this.sandbox.executeSync('bundle exec compensated apply')
+  return this.clientSandbox.executeSync('bundle exec compensated apply')
 })
 
+When(
+  'I run the setup and test scripts for compensated-ruby on each version',
+  { timeout: -1 },
```

Comment:
What does this object `{ timeout: 1 }` do?

Reply:
This [completely disables timeouts](https://github.com/cucumber/cucumber-js/blob/509e64fd5d9835c9101b33a642e6ccac459025b8/docs/support_files/timeouts.md#disable-timeouts) so that the full installation and test suite can run. This task can take 3~5 minutes depending on if folks do or do not have the ruby versions installed already.